src/config/File.cxx

```diff
@@ -72,7 +72,7 @@ config_read_name_value(ConfigBlock &block, char *input, unsigned line)
 		throw FormatRuntimeError("\"%s\" is duplicate, first defined on line %i",
 					 name, bp->line);
 
-	block.AddBlockParam(name, std::move(value), line);
+	block.AddBlockParam(name, value, line);
```

Comment:
Why remote this `std::move()`?

Reply:
warning: std::move of the variable 'value' of the trivially-copyable type 'char *' has no effect; remove std::move() [hicpp-move-const-arg]